www/front_src/src/Resources/Graph/Performance/Legend/Marker.tsx

```diff
@@ -4,26 +4,29 @@ import clsx from 'clsx';
 
 import { makeStyles, Theme } from '@material-ui/core';
 
-const useStyles = makeStyles<Theme, { color?: string }>((theme) => ({
-  icon: {
-    width: 9,
-    height: 9,
-    borderRadius: '50%',
-    marginRight: theme.spacing(1),
-    backgroundColor: ({ color }) => color,
-  },
-  disabled: {
-    color: theme.palette.text.disabled,
-  },
-}));
+const useStyles = makeStyles<Theme, { color?: string; isInTooltip?: boolean }>(
+  (theme) => ({
+    icon: {
+      width: 9,
+      height: ({ isInTooltip }) => (isInTooltip ? 9 : '100%'),
+      borderRadius: ({ isInTooltip }) => (isInTooltip ? '50%' : 0),
+      marginRight: theme.spacing(1),
+      backgroundColor: ({ color }) => color,
+    },
+    disabled: {
+      color: theme.palette.text.disabled,
+    },
+  }),
+);
 
 interface Props {
   disabled?: boolean;
   color: string;
+  isInTooltip?: boolean;
 }
 
-const LegendMarker = ({ disabled, color }: Props): JSX.Element => {
-  const classes = useStyles({ color });
+const LegendMarker = ({ disabled, color, isInTooltip }: Props): JSX.Element => {
```

Comment:
I don’t think we should ever talk about the tooltip in this context, it’s better to add the type (dotted vs bar)